docs/helm-chart-api.md

```diff
@@ -0,0 +1,184 @@
+  
+- GetContent: Get the report summary as a string in either json or yaml format. ReportFormat values are defined and availalble in the reportsummary package:
+    - ```JsonReport``` - for json format.
+    - ```YamlReport``` - for yaml format.
+  
+- SetValues: Used to set value flags to tailor content of the report sumary. Can be used to set a profile vendorType and/or version which by default are the values set in the report.
+
+## Checks
+
+```
+func GetChecks() []CheckName
+```
+
+- GetChecks: Used to get an array of ```CheckName``` types. The array content provide value with can be used for ```verifier.EnableChecks``` and ```verifier.UnEnableChecks```:
+    - ```ChartTesting```
+    - ```ContainsTest```
+    - ```ContainsValuesSchema```
+    - ```ContainsValues```
+    - ```HasKubeVersion```
+    - ```HasReadme```
+    - ```HelmLint```
+    - ```ImagesAreCertified```
+    - ```IsHelmV3```
+    - ```NotContainCsiObjects```
+    - ```NotContainsCRDs```
+    - ```RequiredAnnotationsPresent``` 
+
+
+# Example:
+
+Note: Example does not include error checking for clarity.
+
+1. import the api:
+```
+import (
+	"fmt"
+	"github.com/redhat-certification/chart-verifier/pkg/chartverifier/checks"
+	"github.com/redhat-certification/chart-verifier/pkg/chartverifier/report"
+	"github.com/redhat-certification/chart-verifier/pkg/chartverifier/reportsummary"
+	"github.com/redhat-certification/chart-verifier/pkg/chartverifier/verifier"
+)
+```
+
+2. Create a verifier, set a profile vendortype of redhat, un-enable the chart testing check, and run verify:
+```
+	// Run verify command for a chart, but omit the chart testing check and run checks based on the redhat profile
+	commandSet := make(map[string]interface{})
+	commandSet["profile.vendortype"] = "redhat"
+
+	verifier, verifierErr := verifier.NewVerifier().
+		SetValues(verifier.CommandSet, commandSet).
+		UnEnableChecks([]checks.CheckName{checks.ChartTesting}).
+		Run("https://github.com/redhat-certification/chart-verifier/blob/main/tests/charts/psql-service/0.1.9/psql-service-0.1.9.tgz?raw=true")
+
+```
+3. Get and print, in yaml format, the report created by ```verifier.Verify``` in step 2.
+```
+	// Get and print the report from the verify command
+	report, reportErr := verifier.GetReport().
+		GetContent(report.YamlReport)
+	fmt.Println("report content:\n", report)
+```
+4. Get and print, in json format, a report summary which is based on a report from ```verifier.GetReport``, and using a profile vendor type of partner.
+```
+	// Get and print the report summary  of the report, but using the partnet profile.
+	values := make(map[string]interface{})
+	values["profile.vendortype"] = "partner"
+
+	reportSummary, summmaryErr := reportsummary.NewReportSummary().
+		SetReport(verifier.GetReport()).
+		SetValues(values).
+		GetContent(reportsummary.AllSummary, reportsummary.JsonReport)
+
```

Comment:
Suggestion: Write a sentence or two explaining the outcome and achievement of this procedure that gives a nice closure to this content.